Source/WebCore/page/Page.cpp

```diff
@@ -278,24 +278,24 @@ static Ref<Frame> createMainFrame(Page& page, std::variant<UniqueRef<FrameLoader
 }
 
 Page::Page(PageConfiguration&& pageConfiguration)
-    : m_chrome(makeUnique<Chrome>(*this, WTFMove(pageConfiguration.chromeClient)))
-    , m_dragCaretController(makeUnique<DragCaretController>())
+    : m_chrome(makeUniqueRef<Chrome>(*this, WTFMove(pageConfiguration.chromeClient)))
+    , m_dragCaretController(makeUniqueRef<DragCaretController>())
 #if ENABLE(DRAG_SUPPORT)
-    , m_dragController(makeUnique<DragController>(*this, WTFMove(pageConfiguration.dragClient)))
+    , m_dragController(makeUniqueRef<DragController>(*this, WTFMove(pageConfiguration.dragClient)))
 #endif
-    , m_focusController(makeUnique<FocusController>(*this, pageInitialActivityState()))
+    , m_focusController(std::make_unique<FocusController>(*this, pageInitialActivityState()))
```

Comment:
This should be makeUnique to check that it is fast malloc'ed.

Reply:
done